docker-app/qfieldcloud/core/models.py

```diff
@@ -1764,6 +1764,29 @@ def fallback_output(self) -> str:
                 "The job ended in unknown state. Please verify the project is configured properly, try again and contact QFieldCloud support for more information."
             )
 
+    @property
+    def qgis_version(self) -> str | None:
+        """Returns QGIS app version used for the job.
+        The QGIS version is the one coming from the instanciated worker QGIS app.
+        It is displayed in the job admin.
+        The version number would be in `Major.Minor.Patch-NAME` format, e.g. 3.40.2-Bratislava
```

Comment:
```suggestion

        The QGIS version is the one coming from the instanciated worker QGIS app.
        The version number would be in `Major.Minor.Patch-NAME` format, e.g. 3.40.2-Bratislava

```

I would add a bit of air around the sentences for readability and ignore where it is being shown, we might show it in other places too.